packages/fast-development-site-react/src/components/site/configuration-panel.tsx

```diff
@@ -0,0 +1,213 @@
+import * as React from "react";
+import { toPx } from "@microsoft/fast-jss-utilities";
+import { IDevSiteDesignSystem } from "../design-system";
+import Form from "@microsoft/fast-form-generator-react";
+import { Link, withRouter } from "react-router-dom";
+import { RouteComponentProps } from "react-router";
+import { ComponentViewTypes } from "./component-view";
+import { glyphBuildingblocks, glyphExamples, glyphPage } from "@microsoft/fast-glyphs-msft";
+import ComponentViewToggle from "./component-view-toggle";
+import manageJss, { ComponentStyles, ICSSRules, IJSSManagerProps, IManagedClasses } from "@microsoft/fast-jss-manager-react";
+
+export enum TabType {
+    presets = "Presets"
+};
+
+export interface IConfigurationPanelProps {
+    // onToggleView: () => void;
+    // activeFormData: any;
+    // activeSchema: any;
+    // activeComponentName: string;
+    formChildOptions: any;
+    schema: any;
+    data: any;
+    onChange: any;
+    activeTab?: TabType;
+}
+
+export interface IConfigurationPanelState {
+    activeTab: TabType;
+}
+
+export interface IConfigurationPanelManagedClasses {
+    configurationPanel: string;
+    configurationPanel_controls: string;
+    configurationPanel_controls_tabs: string;
+    configurationPanel_tab: string;
+    configurationPanel_tab__active: string;
+    configurationPanel_tabPanel: string;
+    configurationPanel_paneForm: string;
+}
+
+const style: ComponentStyles<IConfigurationPanelManagedClasses, IDevSiteDesignSystem> = {
+    configurationPanel: {
+        width: "100%"
+    },
+    configurationPanel_controls: {
+        display: "flex",
+        padding: `${toPx(2)} ${toPx(4)}`,
+        "& ul": {
+            margin: "0",
+            padding: "0",
+            listStyleType: "none",
+            "& li": {
+                display: "inline-block"
+            }
+        },
+        "& button": {
+            border: "none",
+            height: toPx(38),
+            padding: `${toPx(6)} ${toPx(8)}`,
+            minWidth: toPx(25),
+            backgroundPosition: "center"
+        }
+    },
+    configurationPanel_controls_tabs: {
+        flexGrow: "1"
+    },
+    configurationPanel_tab: {
+        "& button": {
+            background: "none",
+            "&:focus": {
+                outline: "none"
+            }
+        }
+    },
+    configurationPanel_tab__active: {
+        "& button": {
+            borderRadius: toPx(3),
+            position: "relative",
+            "&:after": {
+                position: "absolute",
+                bottom: "0",
+                left: toPx(4),
+                right: toPx(4),
+                content: "''",
+                height: toPx(2),
+                borderRadius: toPx(3),
+                // TODO: Issue #309 https://github.com/Microsoft/fast-dna/issues/309
+                // background: (config: IDevSiteDesignSystem): string => {
+                //     return config.brandColor;
+                // }
```

Comment:
I believe this todo is out of date as the issue has been closed. You may want to try targeting `::after`.